src/metal.js

```diff
@@ -1,3 +1,21 @@
-// Add Backbone.Metal extensions here
-// 1. Marionette.Class
-// 2. Marionette.Error and Marionette.Deprecate
+// Missing Metal Items
+// 1. Marionette.Class parity
+// 2. Marionette.Deprecate
+
+Marionette.Class = Metal.Class.extend({});
+
+function classify(obj) {
+  return Marionette.Class.extend(
+    _.extend({constructor: obj}, _.omit(obj.prototype, _.keys(Backbone.Events))),
+    _.omit(obj, _.keys(Marionette.Class))
+  );
```

Comment:
You should be able to just do the following since `Backbone.Events` eventually just gets added to `Metal.Class.prototype` directly. I think that will work for `initialize` and `constructor` too. 

```js
_.extend({constructor: obj}, _.omit(obj.prototype, _.keys(Marionette.Class.prototype)));
```